CI/travis/get_ldist

```diff
@@ -0,0 +1,35 @@
+get_ldist() {
+	case "$(uname)" in
+	Linux*)
+		if [ ! -f /etc/os-release ] ; then
+			if [ -f /etc/centos-release ] ; then
+				echo "centos-$(sed -e 's/CentOS release //' -e 's/(.*)$//' \
+					-e 's/ //g' /etc/centos-release)-$(uname -m)"
+				return 0
+			fi
+			ls /etc/*elease
+			[ -z "${OSTYPE}" ] || {
+				echo "${OSTYPE}-unknown"
+				return 0
+			}
+			echo "linux-unknown"
+			return 0
+		fi
+		. /etc/os-release
+		command dpkg --version >/dev/null 2>&1
+		if [ "$?" -ne 0 ] ; then
```

Comment:
nitpick: this could be compacted a bit like
```
if command dpkg --version &> dev/null ; then
     echo $ID-$VERSION_ID-$(dpkg --print-architecture)
else
     echo $ID-$VERSION_ID-$(uname -m)
fi
```

Reply:
that will error on older versions of dash, and cause the script to error out and fail